### PR TITLE
Check and fix grub for net.ifnames=0

### PR DESCRIPTION
**Pull request by atoomic**
https://github.com/cpanel/elevate/issues/190

### CODE REVIEW

**Comment by toddr** [comment]
We shouldn't be doing this unless the new kernel has this option on it.

**Review by toddr — CHANGES_REQUESTED** [review]
I'd rather wait for upstream if they can fix this quickly.

File: elevate-cpanel [hunk 1]
```diff
@@ -1087,6 +1089,27 @@ sub run_stage_5 ($self) {
...
+    # for an autofixer
+    # return unless -e q[/var/cpanel/version/elevate];
+
+    my $grub_conf;
+    {
+        local $/;
+        open( my $fh, '<', DEFAULT_GRUB_FILE ) or return;
+        $grub_conf = <$fh>;
+    }
+
```
Comment (atoomic) [inline elevate-cpanel:1103]: maybe check if `eth0` is here or the output from the leapp log to confirm something like:
```
Detected interface rename eth0 -> ens18.
```

Comment (atoomic) [reply elevate-cpanel:1103]: even better check the output from `/proc/cmdline`

**Pull request merged**